packages/instantsearch.css/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "instantsearch.css",
-  "version": "7.4.5",
+  "version": "7.4.6",
```

Comment:
This should actually be 8.0.0 since there was a breaking change on the vendor prefixes.